packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -3968,6 +3968,7 @@ Map {
       "columns": Array [],
       "hasRowActions": false,
       "hasRowExpansion": false,
+      "hasRowNesting": false,
```

Comment:
@bjornalm Does this new prop need to be added to our docs?

Reply:
It already exists in the Table documentation. The `TableSkeletonWithHeaders.jsx` does not have a mdx file or any dedicated documentation at all. But it is exported so perhaps we should create docs for it? I can add an issue to https://github.com/carbon-design-system/carbon-addons-iot-react/issues/2820 if you want